src/flow.ts

```diff
@@ -228,8 +221,24 @@ export class Flow {
    * @returns an array of Font objects corresponding to the provided `fontNames`.
    */
   static setMusicFont(...fontNames: string[]): Font[] {
+    // HACK during review to introduce the correct stacks step by step
```

Comment:
So after this PR is merged, will this "HACK" go away? What is the correct solution in this method?

Reply:
The hack will stay until all the code is ported this hack avoids that the non ported files produce visual diffrences. In the end it will be:

```
static setMusicFont(...fontNames: string[]): void {
    // Convert the array of font names into an array of Font objects.
    CommonMetrics.fontFamily = fontNames.join(',');
  }
```